content/docs/concepts/performance_validation/index.md

```diff
@@ -11,7 +11,7 @@ Keptn helps automating your tests by having Keptn triggering the test execution
 
 ## Test Automation Process
 
-The following image shows the general process of how Keptn is providing continuous performance verification. An underlying core concept for this process it the SLO/SLI validation as explained in [Quality Gates](../quality_gates/).
+The following image shows the general process of how Keptn is providing continuous performance verification. An underlying core concept for this process it the SLO/SLI-based deployment and release validation as explained in [Quality Gates](../quality_gates/).
```

Comment:
Same as mentioned below, I don't think you can validate a deployment. I'd rather validate a version or a release.

Reply:
I agree with you that promoting/delivering a deployment does not make sense (see your comments above). However, I still see it valid to conduct a deployment validation. Kind of pre-step before releasing the new version. Hence, I would like to leave this as it is.